tests/basic/timeout_variation_10.phpt

```diff
@@ -1,5 +1,7 @@
 --TEST--
 Timeout within shutdown function, variation
+--CREDITS--
```

Comment:
You didn't write this test: If every tiny change of a test lead to someone entering their names in credits section, all tests would soon become unmanageable.